ipatests/test_integration/test_replica_promotion.py

```diff
@@ -570,3 +570,50 @@ def test_replica_install_with_existing_entry(self):
         master.run_command(arg)
 
         tasks.install_replica(master, replica)
+
+
+class TestSubCAkeyReplication(IntegrationTest):
+    """
+    Test if subca key replication is not failing.
+    """
+    topology = 'line'
+    num_replicas = 1
+
+    SUBCA = 'test_subca'
+    SUBCA_CN = 'cn=' + SUBCA
+
+    PKI_DEBUG_PATH = '/var/log/pki/pki-tomcat/ca/debug'
+
+    ERR_MESS = 'Caught exception during cert/key import'
+
+    def test_sub_ca_key_replication(self):
+        master = self.master
+        replica = self.replicas[0]
+
+        result = master.run_command(['ipa', 'ca-add', self.SUBCA, '--subject',
+                                     self.SUBCA_CN])
+        assert result.returncode == 0
+
+        uuid = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
+        auth_id = re.compile('Authority ID: ({})'.format(uuid), re.IGNORECASE)
+        auth_id = "".join(re.findall(auth_id, result.stdout_text))
+
+        cert_nick = '{} {}'.format(IPA_CA_NICKNAME, auth_id)
+
+        # give replication some time
+        time.sleep(30)
+
+        result = replica.run_command(['ipa-certupdate'])
+        assert result.returncode == 0
```

Comment:
You don't need to clutter the code with ``assert result.returncode == 0``. pytest multihost's ``run_command`` method defaults to ``raiseonerr=True``, https://pagure.io/python-pytest-multihost/blob/master/f/pytest_multihost/host.py#_202

Reply:
keep forgetting about this or just like "asserts" :)